bucket4j-spring-boot-starter/src/main/java/com/giffing/bucket4j/spring/boot/starter/filter/reactive/AbstractReactiveFilter.java

```diff
@@ -96,7 +96,7 @@ protected Mono<Void> handleConsumptionProbe(ServerWebExchange exchange, Reactive
 				DataBuffer buffer = exchange.getResponse().bufferFactory().wrap(filterConfig.getHttpResponseBody().getBytes(UTF_8));
 				return response.writeWith(Flux.just(buffer));
 			} else {
-				return Mono.error(new ReactiveRateLimitException("HTTP ResponseBody is null"));
+				return Mono.error(new ReactiveRateLimitException(filterConfig.getHttpStatusCode(), "HTTP ResponseBody is null"));
```

Comment:
If the body is null (else block), you still set the body to "HTTP ResponseBody is null". Wasn't the point of this change to allow empty/null boddies? Using the statuscode from the config is a good addition :)

Reply:
Thanks! The body "HTTP ResponseBody is null" wasn't shown at all. Maybe the ResponseStatusException shouldn't be used in a filter at all. But that is another problem.

Reply:
> Thanks! The body "HTTP ResponseBody is null" wasn't shown at all. Maybe the ResponseStatusException shouldn't be used in a filter at all. But that is another problem.

I think thats because you tested it in the cafeine cache demo project (judging the changed files). That  one uses the servlet filter, not the reactive. Could be the reason why you missed it